docs/02-usage.md

```diff
@@ -1,17 +1,74 @@
+After your device finishes downloading the application, you should be ready to go! 
+
+Before starting make sure to connect the audio output of your device to your Hi-Fi or speaker system. Remember we support multiple [audio interfaces](../docs/audio-interfaces) if you are interested in improving the audio quality of your setup be sure to check it out.
 
 To connect to your balenaSound device:
 * If using Bluetooth: search for your device on your phone or laptop and pair.
 * If using Airplay: select the balenaSound device from your audio output options.
 * If using Spotify Connect: open Spotify and choose the balenaSound device as an alternate output.
 * If using UPnP: select the balenaSound device from your UPnP app like [BubbleUPnP](https://play.google.com/store/apps/details?id=com.bubblesoft.android.bubbleupnp) (Android) or [JuP&P](https://apps.apple.com/app/jup-p-upnp-player-und-fernbedienung/id1069722311) (IOS).
-* The `balenaSound xxxx` name is used by default, where `xxxx` will be the first 4 characters of the device ID in the balenaCloud dashboard.
 
-If you are running in multi-room mode, the device that starts the stream will configure itself as the master and will broadcast a message to all other devices within your balenaCloud application to get them in sync. 
+The `balenaSound <plugin> xxxx` name is used by default, where `xxxx` will be the first 4 characters of the device UUID in the balenaCloud dashboard.
+
+Let the music play!
+
+## Modes of operation
+
+balenaSound supports multiple modes of operation described below:
+
+- Multi-room mode
+- Multi-room client mode
+- Standalone
+
+By default, most devices will start in multi-room mode. You can change that by setting an environment variable, check out the [customization](../docs/customization#general) section to learn how.
+
+**Note:** Multi-room mode is the default mode for most (but not all!) device types. You can read more about default modes [here](../docs/device-support#recommended).
+
+### Multi-room mode
+
+Multi-room mode allows you to play perfectly synchronized audio on multiple devices, it turns balenaSound into a "Sonos-like" multi-room solution. It doesn't matter wether you have 2 or 100 devices, you only need them to be part of the same local network.
+
+When in multi-room mode devices can take one of two roles: 
+- `master`: the device acting as the audio source
+- `client`: any number of devices playing back the audio being sent over by the `master`
+
+Designing a `master` device is easy and requires no configuration. Whenever you start streaming to any device in multi-room mode it will autoconfigure itself to be the `master` device and will broadcast a message to all other devices within your local network to get them in sync. Note that it can take a few seconds for the system to autoconfigure the first time you stream. 
```

Comment:
```suggestion
Designing a `master` device is easy and requires no configuration. Whenever you start streaming to any device in multi-room mode, it will autoconfigure itself to be the `master` device and will broadcast a message to all other devices within your local network to get them in sync. Note that it can take a few seconds for the system to autoconfigure the first time you stream. 
```